build/commands/scripts/commands.js

```diff
@@ -58,6 +58,7 @@ program
   .option('--target_os <target_os>', 'target OS')
   .option('--target_arch <target_arch>', 'target architecture')
   .option('--target_apk_base <target_apk_base>', 'target Android OS apk (classic, modern, mono)', 'classic')
+  .option('--target_output <target_output>', 'target Android output format (apk, aab)', 'apk')
```

Comment:
target_apk_base and target_output aren't relevant to gn_check. Only target_os and target_arch are relevant here because those affect the output dir

Reply:
I would probably leave the default out of this one since we want to do it by build config type

Reply:
I am not sure which one is better, it will probably depend on how CI will be changed. @SergeyZhukovsky did you discuss with Mihai?

Reply:
I will change the default to aab for now but that means local debugging needs an extra option.

Reply:
Maybe @bridiver  meant default to aab only for Release?

Reply:
The change on gn_check is reverted.

Reply:
> Maybe @bridiver meant default to aab only for Release?

I checked the js files, the only place `isOfficialBuild` is checked is when setting channels. I am not sure whether it is a good place to change default though.